### ISSUE TITLE
2.0.0-beta001 release

### ISSUE THREAD

**Issue by blairconrad** [blairconrad/FakeItEasy-annotations#6]
**Ready** when all other issues on this milestone are **Done** and closed.
- [ ] run code analysis in VS in _Release_ mode and address violations (send a regular PR which must be merged before continuing)
- [ ] if necessary, change `VERSION_SUFFIX` on the [CI Server](http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929)
    to appropriate "-beta123" or "" (for non-betas) value and initiate a build
- [ ] check build
-  edit draft release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases):
  - [ ] complete release notes, mentioning non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
  - [ ] attach nupkg
  - [ ] publish the release
- [ ] push NuGet package
- [ ] de-list pre-release or superseded buggy NuGet packages if present
- [ ] update website with contributors list (if in place)
- [ ] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [ ] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [ ] if this is a production release, post links to the NuGet and GitHub release in each issue in this milestone, with thanks to contributors
- if the next release is a production release, then
  - [ ] use `rake set_version[new_version]` to create a pull request that changes the  CommonAssemblyInfo.cs to
       the expected version (of form _xx.yy.zz_), and once that PR has been merged
  - [ ] use `rake create_milestone` to
    - create a new milestone for the next release
    - create a new issue (like this one) for the next release, adding it to the new milestone
    - create a new draft GitHub Release
- if the next release is a pre-production release, then
  - [ ] use `rake create_pre_release[suffix]` to
    - create a new issue (like this one) for the next release
    - create a new draft GitHub Release
  - [ ] change `VERSION_SUFFIX` on the [CI Server](http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929) to the appropriate value (e.g. "-rc01")
- [ ] close this milestone
